### PR TITLE
Bug 1228419 - Added Login Manager flag to AppConstants

### PR DESCRIPTION
**Pull request by sleroux**
Alternative to https://github.com/mozilla/firefox-ios/pull/1373.

Flags live in AppConstants instead of .xcconfig files.

### CODE REVIEW

File: Utils/AppConstants.swift [hunk 1]
```diff
@@ -10,13 +10,28 @@ public enum AppBuildChannel {
...
+ * MOZ_LOGIN_MANAGER - Toggles the Logins manager from appearing in the Settings.
+ *
+ */
+
 public struct AppConstants {
 #if MOZ_CHANNEL_AURORA
     public static let BuildChannel = AppBuildChannel.Aurora
+
+
+    public static let MOZ_LOGIN_MANAGER = false
```
Comment (rnewman) [inline Utils/AppConstants.swift]: This is fine, but in the future you might consider inverting these, which is what we do on Android:

```
public static let MOZ_LOGIN_MANAGER =
#if MOZ_CHANNEL_AURORA
false
#else
…
#endif
```

When these flags proliferate you increase the chances of you writing code that doesn't compile with some sets of flags due to omitted or misnamed vars in one of these branches.

Comment (sleroux) [inline Utils/AppConstants.swift]: Just saw this in the Android code base - much nicer. I'll update the PR.

**Comment by sleroux** [comment]
I was able to get it to work but had to wrap the value in a closure which is fine I guess.

**Comment by rnewman** [comment]
If you use AppConstants in your AppConstants definition you don't…

**Pull request merged**